cmake/Modules/Findfmt.cmake

```diff
@@ -0,0 +1,64 @@
+add_library(fmt INTERFACE IMPORTED)
+
+set(fmt_external_project_name fmtlib_fmt)
+
+find_path(fmt_INCLUDE_DIR fmt/format.h
+  PATHS ${EP_PREFIX}/src/${fmt_external_project_name}/include)
```

Comment:
Not really a good idea, since the user might want to use system-wide installed library

Reply:
According to [the docs](https://cmake.org/cmake/help/latest/command/find_path.html), the system standard paths are searched as well.